frontends/p4/typeMap.cpp

```diff
@@ -50,6 +50,16 @@ bool TypeMap::isCompileTimeConstant(const IR::Expression* expression) const {
     return result;
 }
 
+void TypeMap::setLeftValue(const IR::Expression* expression,
```

Comment:
If I add the following comment above the method, would that work?

```
// Method sets left value of expression arg if orig arg
// is a left value.
```

Or I  can call the function `copyLeftValue` because if orig is a leftValue, then the expression is also added to the leftValues list.